Automattic/jetpack
# Bump default node version to 18.17.0 to match Calypso

## Proposed changes:

Updating `.nvmrc` and `version.sh`. Leaving the engines in `package.json` at 18.13.0 as we're not planning on using any of the thing added in versions in between.

### Other information:


## Jetpack product discussion


p4TIVU-aKr-p2
Announcement for when this is merged: pdWQjU-tK-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Check that stuff works.
  * If you use nvm, see that `nvm use` gets the right version.
  * You might also check the output from `jetpack` and `tools/check-development-environment.sh` if you have a lower version of node 18 installed.
